contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -170,11 +175,15 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       ((gasPriceWei * s_config.fulfillmentGasPriceOverEstimationBP) / 10_000);
     /// @NOTE: Basis Points are 1/100th of 1%, divide by 10_000 to bring back to original units
 
-    uint96 juelsPerGas = _getJuelsPerGas(gasPriceWithOverestimation);
-    uint256 estimatedGasReimbursement = juelsPerGas * executionGas;
-    uint96 fees = uint96(donFee) + uint96(adminFee);
+    uint256 executionGas = s_config.gasOverheadBeforeCallback + s_config.gasOverheadAfterCallback + callbackGasLimit;
+    uint256 executionWei = gasPriceWithOverestimation * executionGas;
+    uint256 l1FeeWei = ChainSpecificUtil._getCurrentTxL1GasFees(msg.data);
+
+    uint96 estimatedGasReimbursement = _getJuelsFromWei(executionWei + l1FeeWei);
```

Comment:
How about call it estimatedGasReimbursementJuels and also inline executionWei calculation inside it to avoid an extra variable?